README.md

```diff
@@ -1,6 +1,8 @@
 zlocker: zookeeper based isolated command execution
 ===================================================
 
+:warning: Usage of this tool is deprecated and being replaced by `consul lock`.
```

Comment:
Shall we add an ETA and a link to a relevant documentation page explaining how to use/configure `consul lock`?